app/models/dj_metrics.rb

```diff
@@ -65,14 +62,14 @@ def refresh_queue_sizes!
     Delayed::Job.where('failed_at IS NULL').where('locked_by IS NULL').group(:queue).count.each do |queue, size|
```

Comment:
Is there a reason we wouldn't use:?
```suggestion
    Delayed::Job.where(failed_at: nil, locked_by: nil).group(:queue).count.each do |queue, size|
```